stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/models/identifier/Identifiers.kt

```diff
@@ -224,6 +224,19 @@ internal fun HideChannelIdentifier(
     return result
 }
 
+/**
+ * Identifier for a [ChatClient.getMessage] call.
+ */
+@Suppress("FunctionName", "MagicNumber")
```

Comment:
Change made when adding OP to `ChatClient.getMessage()` in `v5`.

It's non-conflicting with the one made in the `v6` PR seen here: https://github.com/GetStream/stream-chat-android/pull/4635/files#diff-ef661b55135a2b5ebd91e17fe89f833aaf5dd222d11d1e917f5ab17cc9878f0fR227